### PR TITLE
add genz-109 palindrome struct

### PR DESCRIPTION
**Pull request by aljab017**

### CODE REVIEW

File: examples/maather_work_area/genz-109.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'''
+A function that builds a palindrome structure
```
Comment (CodelineAtyab) [inline examples/maather_work_area/genz-109.py:2]: Please consider updating the multiline comment to show what a palindrome structure is. Is it a triangle. If yes, how does it look like and an example input output would help devs understand what this module is doing.

File: examples/maather_work_area/genz-109.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+'''
+A function that builds a palindrome structure 
+''' 
+def palindrome_structure(n):
```
Comment (CodelineAtyab) [inline examples/maather_work_area/genz-109.py:4]: I believe, n can be changed to num_of_rows_for_triangle

File: examples/maather_work_area/genz-109.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    temp = 0
```
Comment (CodelineAtyab) [inline examples/maather_work_area/genz-109.py:5]: Consider adding a single line comment to describe what this temp is used for

**Review by CodelineAtyab — CHANGES_REQUESTED** [review]
Please respond to comments

**Review by Sughiya-AlSaid — DISMISSED** [review]

**Review by Abbas-Abdulrab — APPROVED** [review]

**Review by CodelineAtyab — APPROVED** [review]
Great!

**Review by Rudainasaleh — APPROVED** [review]

**Pull request merged**